receiver/vcsmetricsreceiver/README.md

```diff
@@ -31,7 +36,7 @@ The collection interval is common to all scrapers and is set to 30 seconds by de
 > and organizations. You may run into significantly more rate limiting.
 
 ```yaml
-gitprovider:
+vcsmetrics:
```

Comment:
This README has a link below to the Liatrio distribution, just a couple comments:

1. I believe the referred to distribution is `downstream` from `contrib`, as it relies on `contrib`, not vice versa.
2. My actual comment, the referenced config file will need to be updated to account for this name change.

In general I'd prefer to have configuration examples in this repository to avoid stale information. It's a pretty common issue to forget to update READMEs with updated configuration information, and it will only be harder to keep all examples and information up to date if there are more places to keep updated. I'd appreciate input though, maybe there's a good reason to keep location of this information the way it is.

Reply:
@crobert-1  - This pr is just a move, but I'm happy to update the docs minimally where needed in this PR. 

The mention of the Liatrio distro is there because the component has largely been developed there first, aside for this rename. Being that, there are features in that distro (like a GitLab scraper, and CVE Metrics from GitHub) that haven't been contributed here yet. That being said, that one sentence can be removed if necessary. IIRC (it's been almost a year now) this was added intentionally. Especially for those that want to use the component and don't build their own distros since this hasn't been built into contrib yet. Of course, this was a different time because back then these types of distros were supported as being linked directly as a distro in the generated docs. 

The config examples in the README are up-to-date. Happy to remove that mention though. I would suggest eventually adding another repository we have that runs all this locally through Kubernetes for those that want to get running locally quickly too. 

I did say "largely" as of late I've been doing most of these contributions on my own time. These VCS updates haven't made it into the Liatrio distro as of yet. I could go either way, so if it's important to remove I can align.

Reply:
Thanks for providing the extra context, I wasn't familiar with the history here 👍 In that case, it's okay with me to leave it as-is, the reference makes sense in this case. As long as we can keep the references up to date to avoid conflicting information it's fine with me.